index.html

```diff
@@ -11,15 +11,21 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
         rel="stylesheet">
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
 </head>
 
 <body>
-    <header id="Portfolio">
+    <header id="Portfolio" class="parent">
         <h2 class="log"><a class="Welcome" href="#">Welcome</a></h2>
         <nav class="navbar-humberger">
-            <i class="fa fa-bars " aria-hidden="true"></i>
+            <i class="menuIcon material-icons">menu</i>
         </nav>
         <ul class="navbar-links">
+            <li>
+            <i class="closeIcon material-icons d-none">close</i>
```

Comment:
- [ ] The hamburger button and menu should follow the design from the chosen template in Figma (e.g., font, colors, images, tests, margins). => this close button should only be shown when the mobile menu is visible, kindly address this issue

- [ ] [Optional] This is just an optional requirement, don't take it hard you can decide to implement it depending on whether you have bandwidth for it or not: the mobile menu should disappear on desktop screens, currently when the screen grows, it remains in the screen. 

Kindly watch the following screencast for learning more context of my two comments above

https://user-images.githubusercontent.com/56886064/145965795-382f13fb-f9da-44d4-86e4-f61402e5b333.mp4

Reply:
thanks for this feedback